### PR TITLE
Add .github

### PR DESCRIPTION
**Pull request by blase147**
This is the "Hello Microverse Project" for beginner Micronauts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,36 @@
...
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
+- GitHub: [@blase147](https://github.com/blase147)
+- Twitter: [@ChukwumaMosanya](https://twitter.com/ChukwumaMosanya)
+- LinkedIn: [ChukwumaMosanya](www.linkedin.com/in/chukwuma-mosanya-346453)
```
Comment (mwanawabangona) [inline README.md:18]: - [ ] good so far 
- [ ] on this section that where you adding people you are saluting or who helped you through the project. 
- [ ] kindly note the you don't add the author here

File: README.md [hunk 2]
```diff
@@ -1,75 +1,36 @@
...
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (mwanawabangona) [inline README.md]: - [ ] kindly note here that where you add the name of the author and there social handles.
- [ ] please make amends

File: README.md [hunk 3]
```diff
@@ -1,75 +1,36 @@
...
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+## Author
```
Comment (mwanawabangona) [inline README.md]: - [ ] Here is where you explain how the person can clone and the project on there machine. please see the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for further understandings

File: README.md [hunk 4]
```diff
@@ -1,75 +1,36 @@
...
 
-> Description the project.
+> A Microverse project for beginer micronauts.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
```
Comment (mwanawabangona) [inline README.md:12]: - [ ] here you write the languages you used to build the project

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] going through the  `readme` [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for further understanding

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @blase147, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**